openvdb/openvdb/unittest/TestTree.cc

```diff
@@ -2755,3 +2756,204 @@ TEST_F(TestTree, testNodeCount)
     EXPECT_EQ(tree.leafCount(), nodeCount2.front());// leaf nodes
     for (size_t i=0; i<nodeCount2.size(); ++i) EXPECT_EQ( nodeCount1[i], nodeCount2[i]);
 }
+
```

Comment:
I think this is a diffing error - I've moved these methods to TestRootNode and TestInternalNode now.

Reply:
Thanks for finding this! Should be fixed now.